<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MorsMainWindow</class>
 <widget class="QMainWindow" name="MorsMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1296</width>
    <height>574</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MorsMainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1296</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_As"/>
    <addaction name="actionClose"/>
    <addaction name="separator"/>
    <addaction name="actionPrint"/>
    <addaction name="actionOption"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionRead_Only"/>
    <addaction name="actionPreView"/>
    <addaction name="actionEdit_Only"/>
    <addaction name="separator"/>
    <addaction name="actionFull_Screen"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionFlush"/>
    <addaction name="actionView_HTML_Source"/>
    <addaction name="separator"/>
    <addaction name="actionCopy"/>
    <addaction name="actionCut"/>
    <addaction name="actionPlast"/>
    <addaction name="separator"/>
    <addaction name="actionRedo"/>
    <addaction name="actionUndo"/>
    <addaction name="separator"/>
    <addaction name="actionFind"/>
   </widget>
   <widget class="QMenu" name="menuInsert">
    <property name="title">
     <string>Insert</string>
    </property>
    <addaction name="actionImage"/>
    <addaction name="actionLink"/>
    <addaction name="actionMathJax"/>
   </widget>
   <widget class="QMenu" name="menuFormat">
    <property name="title">
     <string>Format</string>
    </property>
    <addaction name="actionBlob"/>
    <addaction name="actionItalic"/>
    <addaction name="actionUnder_Line"/>
    <addaction name="separator"/>
    <addaction name="actionBigger"/>
    <addaction name="actionSmaller"/>
    <addaction name="separator"/>
    <addaction name="actionSub"/>
    <addaction name="actionSup"/>
    <addaction name="separator"/>
    <addaction name="actionCode"/>
    <addaction name="actionQuote"/>
    <addaction name="separator"/>
    <addaction name="actionAlign_Left"/>
    <addaction name="actionAlign_Right"/>
    <addaction name="actionAlign_Center"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout_Qt"/>
    <addaction name="actionAbout_MorsMark"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuView"/>
   <addaction name="menuEdit"/>
   <addaction name="menuInsert"/>
   <addaction name="menuFormat"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionNew">
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionSave_As">
   <property name="text">
    <string>Save As</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>Close</string>
   </property>
  </action>
  <action name="actionPrint">
   <property name="text">
    <string>Print</string>
   </property>
  </action>
  <action name="actionOption">
   <property name="text">
    <string>Option</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionRead_Only">
   <property name="text">
    <string>Read Only</string>
   </property>
  </action>
  <action name="actionPreView">
   <property name="text">
    <string>PreView</string>
   </property>
  </action>
  <action name="actionEdit_Only">
   <property name="text">
    <string>Edit Only</string>
   </property>
  </action>
  <action name="actionFull_Screen">
   <property name="text">
    <string>Full Screen</string>
   </property>
  </action>
  <action name="actionFlush">
   <property name="text">
    <string>Flush</string>
   </property>
  </action>
  <action name="actionView_HTML_Source">
   <property name="text">
    <string>View HTML Source</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="text">
    <string>Copy</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="text">
    <string>Cut</string>
   </property>
  </action>
  <action name="actionPlast">
   <property name="text">
    <string>Paste</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="text">
    <string>Redo</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="text">
    <string>Undo</string>
   </property>
  </action>
  <action name="actionFind">
   <property name="text">
    <string>Find</string>
   </property>
  </action>
  <action name="actionImage">
   <property name="text">
    <string>Image</string>
   </property>
  </action>
  <action name="actionLink">
   <property name="text">
    <string>Link</string>
   </property>
  </action>
  <action name="actionMathJax">
   <property name="text">
    <string>MathJax</string>
   </property>
  </action>
  <action name="actionAbout_Qt">
   <property name="text">
    <string>About Qt</string>
   </property>
  </action>
  <action name="actionAbout_MorsMark">
   <property name="text">
    <string>About MorsMark</string>
   </property>
  </action>
  <action name="actionBlob">
   <property name="text">
    <string>Blob</string>
   </property>
  </action>
  <action name="actionItalic">
   <property name="text">
    <string>Italic</string>
   </property>
  </action>
  <action name="actionUnder_Line">
   <property name="text">
    <string>Under Line</string>
   </property>
  </action>
  <action name="actionBigger">
   <property name="text">
    <string>Bigger</string>
   </property>
  </action>
  <action name="actionSmaller">
   <property name="text">
    <string>Smaller</string>
   </property>
  </action>
  <action name="actionSub">
   <property name="text">
    <string>Sub</string>
   </property>
  </action>
  <action name="actionSup">
   <property name="text">
    <string>Sup</string>
   </property>
  </action>
  <action name="actionCode">
   <property name="text">
    <string>Code</string>
   </property>
  </action>
  <action name="actionQuote">
   <property name="text">
    <string>Quote</string>
   </property>
  </action>
  <action name="actionAlign_Left">
   <property name="text">
    <string>Align Left</string>
   </property>
  </action>
  <action name="actionAlign_Right">
   <property name="text">
    <string>Align Right</string>
   </property>
  </action>
  <action name="actionAlign_Center">
   <property name="text">
    <string>Align Center</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
